deploy/base/deployment.yml

```diff
@@ -45,16 +45,49 @@ spec:
             initialDelaySeconds: 5
             periodSeconds: 5
             timeoutSeconds: 1
-          env:
-            - name: HAWTIO_ONLINE_RBAC_ACL
-              value: /etc/hawtio/rbac/ACL.yaml
           resources:
             requests:
               cpu: "0.2"
               memory: 32Mi
             limits:
               cpu: "1.0"
               memory: 500Mi
+          volumeMounts:
+            - name: hawtio-online-tls-serving
+              mountPath: /etc/tls/private/serving
+        - name: hawtio-online-gateway
+          image: quay.io/hawtio/online-gateway
+          env:
+            - name: HAWTIO_ONLINE_RBAC_ACL
+              value: /etc/hawtio/rbac/ACL.yaml
+            - name: HAWTIO_ONLINE_GATEWAY_APP_PORT
+              value: "3443"
+            - name: HAWTIO_ONLINE_GATEWAY_WEB_SERVER
+              value: https://localhost:8443
+            - name: HAWTIO_ONLINE_GATEWAY_SSL_KEY
+              value: /etc/tls/private/serving/tls.key
+            - name: HAWTIO_ONLINE_GATEWAY_SSL_CERTIFICATE
+              value: /etc/tls/private/serving/tls.crt
+            - name: HAWTIO_ONLINE_GATEWAY_SSL_CERTIFICATE_CA
+              value: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+          ports:
+            - name: express
+              containerPort: 3443
```

Comment:
is the app port also https? otherwise, 3443 is a bit misleading. maybe you can just use 3000?